src/components/MapEditorView/MapEditorView.tsx

```diff
@@ -177,7 +177,6 @@ export const MapEditorView: React.FC<MapEditorViewProps> = ({
           initialItems={gridItems}
           updateItems={updateItems}
           initialArrowItems={arrowItems}
```

Comment:
Do we need `initialArrowItems`?

Reply:
yes, in order to edit arows, we need to know which exist already.